### PR TITLE
Implement VPC CRUD

### PR DESCRIPTION
**Pull request by Khyme**
- https://github.com/timescale/forge-issues/issues/777

### CODE REVIEW

File: examples/resources/vpcs.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+terraform {
```
Comment (alevy713) [inline examples/resources/vpcs.tf:1]: slight filepath change to get this on the registry:

```
examples/resources/timescale_vpcs/resource.tf
```

[reference docs](https://github.com/hashicorp/terraform-provider-scaffolding-framework/tree/main/examples)

**Review by alevy713 — COMMENTED** [review]
Looks like all the peering connection bits are commented out. Should those be included?

File: internal/client/vpc.go [hunk 2]
```diff
@@ -40,6 +43,14 @@ type VpcsResponse struct {
 	Vpcs []*VPC `json:"getAllVpcs"`
 }
 
+type CreateVpcResponse struct {
+	Vpc *VPC `json:"createVpc"`
```
Comment (alevy713) [inline internal/client/vpc.go:47]: nit: VPC per [go wiki ](https://go.dev/wiki/CodeReviewComments#initialisms)

**Comment by Khyme** [comment]
@alevy713 good comment on the peering connections and yes it's deliberate. This PR only adds the create/import/update/delete vpc methods, of the vpc "shell" in our system.
the peering connection is not added yet (there's a customer request in the form of an issue in the repo for it too btw!) but the model mapping is not so trivial.
I plan to do this in a separate PR to make things a bit lighter/easier to review.
Also I still need to add tests in this one.

File: internal/client/client.go [hunk 3]
```diff
@@ -81,7 +93,7 @@ func getURL(env string) string {
 	if value, ok := os.LookupEnv("TIMESCALE_DEV_URL"); ok {
 		return value
 	}
-	return "https://console.cloud.timescale.com/api/query"
+	return "https://api.dev.metronome-cloud.com/api/query"
```
Comment (itroot) [inline internal/client/client.go:96]: I think it should probably not go to the release

**Review by itroot — APPROVED** [review]

**Pull request merged**